src/sync/remote_mongo_collection.hpp

```diff
@@ -20,18 +20,22 @@
 #define REMOTE_MONGO_COLLECTION_HPP
 
 #include "sync/app_service_client.hpp"
-#include <realm/util/optional.hpp>
+#include "sync/generic_network_transport.hpp"
+#include "util/bson/bson.hpp"
```

Comment:
Newline before angled imports (I know it wasn't there before, but it should have been)

Reply:
Done